src/CqlBuilderPanel/definitionsSection/DefinitionsSection.test.tsx

```diff
@@ -2,40 +2,107 @@ import * as React from "react";
 import { render, screen, waitFor, act } from "@testing-library/react";
 import DefinitionsSection from "./DefinitionsSection";
 import userEvent from "@testing-library/user-event";
-import { CqlBuilderLookupData } from "../../model/CqlBuilderLookup";
+import { CqlBuilderLookup, Lookup } from "../../model/CqlBuilderLookup";
 
 describe("DefinitionsSection", () => {
+  const tjcMeasurementPeriod = {
```

Comment:
looks same as  `src/CqlBuilderPanel/__mocks__/MockCqlBuilderLookupsTypes.ts`. one of this can be deleted?